### DISCUSSION TITLE
PHP 7.4 + 3rd party extensions for Debian Bookworm

### DISCUSSION THREAD

**Discussion by Whissi** [oerdnj/deb.sury.org#1961] [Q&A]
Hi,

I noticed that there is already https://packages.sury.org/php/dists/bookworm/ which contains basic php7.4-* packages for Bookworm which is great, thank you!

However, it looks like packages for 3rd party extensions like

- php7.4-amqp
- php7.4-ssh2
- php7.4-xdebug

are missing. Are you going to add them as well?

**Comment by oerdnj** [comment]
Yup, those are packages without any new release since I added support for Bookworm couple weeks ago. I’ll trigger the rebuild shortly.
